test/test-signal.c

```diff
@@ -22,6 +22,28 @@
 #include "uv.h"
 #include "task.h"
 
+#define MAX_PID 32768
+TEST_IMPL(kill_invalid_signum) {
+  int pid;
+
+  /* Find unused pid. */
+  pid = 1;
+  while (pid < MAX_PID && uv_kill(pid, 0) != UV_ESRCH)
+    ++pid;
```

Comment:
This is racy as heck - the pid can get used while the test is in progress.  I'm okay with making it a Windows-only test if that is easier for you.